client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/package-row.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { map, uniq } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ItemRow, { Header as ItemRowHeader } from './item-row';
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import Dropdown from 'woocommerce/woocommerce-services/components/dropdown';
+import Checkbox from 'woocommerce/woocommerce-services/components/checkbox';
+import {
+	setContentsType,
+	setContentsExplanation,
+	setRestrictionType,
+	setRestrictionExplanation,
+	setAbandonOnNonDelivery,
+	setITN,
+} from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+
+const PackageRow = ( props ) => {
+	const {
+		siteId,
+		orderId,
+		errors,
+		packageId,
+		translate,
+		contentsType,
+		contentsExplanation,
+		restrictionType,
+		restrictionExplanation,
+		abandonOnNonDelivery,
+		itn,
+		items,
+	} = props;
+	const abandonHandler = () => props.setAbandonOnNonDelivery( ! abandonOnNonDelivery );
+
+	return <div className="customs-step__package">
+		<label htmlFor={ packageId + '_abandonOnNonDelivery' }>
+			<Checkbox
+				id={ packageId + '_abandonOnNonDelivery' }
+				checked={ ! abandonOnNonDelivery }
+				onChange={ abandonHandler } />
+			<span>{ translate( 'Return to sender if package is unable to be delivered' ) }</span>
```

Comment:
I found the phrasing confusing, how about "Send the package back if the package is undeliverable" or something similar with "send the package back"

Reply:
I thought `"Return to sender"` is a common idiom, at least in the US. Can I get a US-Hydra fact-check?